src/control/cmd/daos/pretty/pool.go

```diff
@@ -72,6 +72,7 @@ func PrintPoolInfo(pi *daos.PoolInfo, out io.Writer) error {
 	w := txtfmt.NewErrWriter(out)
 
 	// Maintain output compatibility with the `daos pool query` output.
+	// Pool state should only show degraded when rebuild is active.
```

Comment:
No longer accurate with the latest incarnation of this PR. Should change the title/description of the PR as well, to reflect the new approach. This approach is fine with me, though, and I think it removes the ambiguity of the term "degraded".